src/__test__/Home.test.js

```diff
@@ -0,0 +1,61 @@
+import { render } from '@testing-library/react';
+import { useSelector } from 'react-redux';
+import Products from '../Components/Products/Products';
+import ProductsPage from '../Components/Products/ProductsPage';
+import store from '../Redux/configureStore';
+
+jest.mock('react-redux');
+jest.mock('react-router-dom');
+
```

Comment:
- [ ] As mentioned by the previous reviewer, kindly make sure you add **unit tests for some pure functions** in your Redux code. It is part of the requirements